script.py

```diff
@@ -0,0 +1,239 @@
+"""To use this script, install the provided requirements.txt file and
+supply a secrets.py file with the required parameters listed below.
+
+JSON records will be written into the specified file and HTML reports
+will be written into an outputs filter under the provided directory.
+"""
+
+
+from datetime import datetime, timedelta
+from jinja2 import Template
+from ua_parser import user_agent_parser
+
+import collections
+import json
+import os
+import requests
+import secrets
+import smtplib
+
+
+# This is the absolute path to the directory where you'd like
+# the JSON and HTML files to be written.
+# The template HTML file should also be placed in this
+# directory.
+DIR = secrets.DIR
+# This is the name of the JSON file that you'd like to write to
+FILENAME = secrets.FILENAME
+# This is the username for your LA1 login
+USERNAME = secrets.USERNAME
+# This is the password for your LA1 login
+PASSWORD = secrets.PASSWORD
+# Email address from which you'd like to send viewer information
+FROM_EMAIL = secrets.FROM_EMAIL
+# Email address to which you'd like to send viewer information
+TO_EMAIL = secrets.TO_EMAIL
+
+
+def send_email(event):
+    server = smtplib.SMTP("aspmx.l.google.com", 25)
```

Comment:
Maybe we should allow configuration  of this? I guess it's a pain to support every possible SMTP auth situation. Maybe we just throw a note in the readme that it uses Google's anonymous SMTP and therefore can only be sent to Gmail/Gsuite users? If someone needs something more complicated, then "PRs accepted." ;)

Reply:
Yeah I think I'll leave a comment there and mention it in the README when I write that